experimental/hip/dynamic_symbol_tables.h

```diff
@@ -33,13 +31,23 @@ IREE_HIP_PFN_STR_DECL(hipGetErrorName, hipError_t)
 IREE_HIP_PFN_STR_DECL(hipGetErrorString, hipError_t)
 IREE_HIP_PFN_DECL(hipHostFree, void *)
 IREE_HIP_PFN_DECL(hipHostGetDevicePointer, void **, void *, unsigned int)
+IREE_HIP_PFN_DECL(hipHostMalloc, void **, size_t, unsigned int)
+IREE_HIP_PFN_DECL(hipHostRegister, void *, size_t, unsigned int)
+IREE_HIP_PFN_DECL(hipHostUnregister, void *)
 IREE_HIP_PFN_DECL(hipInit, unsigned int)
 IREE_HIP_PFN_DECL(hipMalloc, void **, size_t)
+IREE_HIP_PFN_DECL(hipMallocFromPoolAsync, void **, size_t, hipMemPool_t,
+                  hipStream_t)
 IREE_HIP_PFN_DECL(hipMallocManaged, hipDeviceptr_t *, size_t, unsigned int)
-IREE_HIP_PFN_DECL(hipMemAllocHost, void **, size_t, unsigned int)
 IREE_HIP_PFN_DECL(hipMemcpy, void *, const void *, size_t, hipMemcpyKind)
 IREE_HIP_PFN_DECL(hipMemcpyAsync, void *, const void *, size_t, hipMemcpyKind,
                   hipStream_t)
+IREE_HIP_PFN_DECL(hipMemPrefetchAsync, const void *, size_t, int, hipStream_t)
```

Comment:
This one is not alphabetically sorted? :)

Reply:
Ah missed it, thanks! :)